p2p/protocol/identify/id_delta.go

```diff
@@ -26,7 +25,7 @@ func (ids *IDService) deltaHandler(s network.Stream) {
 		return
 	}
 
-	defer helpers.FullClose(s)
+	defer s.Close()
```

Comment:
what qualifies as requiring a reset? e.g. does failing to consume the delta warrant a reset?

Reply:
Yes. Although the other end likely isn't listening for an error.